### ISSUE TITLE
Parallelization Bug: multi_scan returns false regardless of whether or not a parallel run passed or failed

### ISSUE THREAD

**Issue by lyndsey-ferguson** [lyndsey-ferguson/fastlane-plugin-test_center#117]
### New Issue Checklist

- [x] Updated `fastlane-plugin-test_center` to the latest version
- [x] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [x] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [x] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

### Issue Description

Each parallel test run returns false regardless of whether or not a parallel test run succeeded or not. I tweaked the code to print out the results of an individual test worker. It returns false for both success and failure.

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>

### Environment

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>

**Comment by lyndsey-ferguson** [comment]
@yousefhamza @Vkt0r can you test this code to check if all test runs pass, multi_scan reports a pass? 

If so, can you modify your `Pluginfile` per my instructions below, run `bundle install`, and then run your fastlane again?

Pluginfile:
```
gem 'fastlane-plugin-test_center', :git => "https://github.com/lyndsey-ferguson/fastlane-plugin-test_center.git", :branch => "117-fix-parallel-test-runs-only-return-false"
```

Please attach the console output as a text file to this issue (makes it easier for me to review).

**Comment by lyndsey-ferguson** [comment]
Don't forget to test this please and thank you. 🙏 

I've tested it with both a parallelized run of tests that all pass to make sure it returns true. Same with some that fail returning false.

**Comment by Vkt0r** [comment]
@lyndsey-ferguson No no don't worry I'll test it in the project from I got it (sorry different machine 😅). I'll throw results tomorrow.

**Comment by lyndsey-ferguson** [comment]
@yousefhamza @Vkt0r don't forget about this test-request
# 💫

**Comment by Vkt0r** [comment]
@lyndsey-ferguson Sorry for the delay, I've been testing it during the last couple of days and it's working fine on my side.

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.8.0.parallelizing.beta.9](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.8.0.parallelizing.beta.9)
